pkg/machine/config.go

```diff
@@ -72,10 +72,10 @@ var (
 
 type Download struct {
 	Arch                  string
-	Artifact              string
```

Comment:
Why is this public, but the type is private?

Reply:
baby steps ... trying to not export everything; only as needed.